updatesnap/updatesnapyaml.py

```diff
@@ -40,9 +40,16 @@ def get_working_branch(self, project_url):
                 break
         return working_branch
 
-    def get_yaml_file(self, project_url):
+    def get_yaml_file(self, project_url, yaml_path):
         """ Searches in a project for the 'snapcraft.yaml' file and
             returns its contents """
+        if yaml_path != 'None':
```

Comment:
And, of course, this should be changed to `if yaml_path is not None:` after doing the change in the default value.

Reply:
Done refracting code